flow/peerdbenv/config.go

```diff
@@ -92,3 +92,17 @@ func PeerDBAlertingEmailSenderRegion() string {
 func PeerDBAlertingEmailSenderReplyToAddresses() string {
 	return GetEnvString("PEERDB_ALERTING_EMAIL_SENDER_REPLY_TO_ADDRESSES", "")
 }
+
+func PeerDBCurrentEncKeyID() string {
+	return GetEnvString("PEERDB_CURRENT_ENC_KEY_ID", "")
+}
+
+func PeerDBEncKeys() shared.PeerDBEncKeys {
+	return GetEnvJSON("PEERDB_ENC_KEYS", make(shared.PeerDBEncKeys, 0))
```

Comment:
```suggestion
	return GetEnvJSON[shared.PeerDBEncKeys]("PEERDB_ENC_KEYS", nil)
```

Reply:
i liked it before because i could invoke `.Get` on it without worrying about NPE.

Reply:
right, was thinking in terms of `map`